cypress/e2e/signIn.cy.js

```diff
@@ -28,7 +34,21 @@ describe('Sign In page', () => {
     homePage.assertHeaderContainUsername(user.username);
   });
 
-  it('should not provide an ability to log in with wrong credentials', () => {
+  it('shouldn\'t provide an ability to log in with wrong email', () => {
+    signInPage.typeEmail(wrongEmail);
+    signInPage.typePassword(user.password);
+
+    signInPage.clickSignInBtn();
+
+    signInPage.assertLoginFailed();
+  });
+
+  it('shouldn\'t provide an ability to log in with wrong password', () => {
+    signInPage.typeEmail(user.email);
+    signInPage.typePassword(wrongPassword);
+
+    signInPage.clickSignInBtn();
 
+    signInPage.assertLoginFailed();
```

Comment:
The `assertLoginFailed` method should accurately verify that the login attempt has failed. Ensure this method checks for an error message or any other indication of a failed login.